interfaces/Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -545,16 +545,15 @@ export class ProgramsServiceApiService {
     return res;
   }
 
-  public async toggleBlockWallet(
+  public async pauseCard(
     programId: number,
+    referenceId: string,
     tokenCode: string,
     block: boolean,
   ): Promise<any> {
-    return await this.apiService.post(
+    return await this.apiService.patch(
       environment.url_121_service_api,
-      `/programs/${programId}/financial-service-providers/intersolve-visa/wallets/${tokenCode}/${
-        block ? 'block' : 'unblock'
-      }`,
+      `/programs/${programId}/registrations/${referenceId}/financial-service-providers/intersolve-visa/cards/${tokenCode}?pause=${block}`,
```

Comment:
Small thing for making it cleaner: IMO we should not use the term "block" here anymore. Maybe just {pause}? I am not familiar with the context in which this is called, so not sure on the best name.